xcape/test/test_core.py

```diff
@@ -44,7 +44,7 @@ def test_calc_cape_shape_3d(p_t_td_3d, p_t_td_surface, sourcein, n_returns, use_
     if vertical_levin=='sigma':
         args = (p, t, td, ps, ts, tds)
     elif vertical_levin =='pressure':        
-        args = (p, t, td, ps, ts, tds,ps*0+1)
+        args = (np.ones(t.shape[-1]), t, td, ps, ts, tds)
```

Comment:
pres_lev_pos is not an argument anymore
and p for pressure levels need to be 1d